qos-service-impl/src/main/java/com/palantir/atlasdb/qos/ratelimit/QosRateLimiters.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.qos.ratelimit;
+
+import org.immutables.value.Value;
+
+import com.palantir.atlasdb.qos.config.QosLimitsConfig;
+
+@Value.Immutable
+public interface QosRateLimiters {
+
+    static QosRateLimiters create(QosLimitsConfig config, long maxBackoffSleepTimeMillis) {
+        QosRateLimiter readLimiter = QosRateLimiter.create(maxBackoffSleepTimeMillis);
+        readLimiter.updateRate(config.readBytesPerSecond());
```

Comment:
It's weird that this is a setter, and not a parameter of the create method... But I'm not sure if we plan to create the QosRateLimiter somewhere else (where we'd need Double.MAX_VALUE), so maybe this is fine. If this is the sole place we create it, we could add it as a parameter of the factory method.

Reply:
the setter exists so the rate can be adjusted dynamically